OPCUA/opcua/OPCUA_01.cmd

```diff
@@ -0,0 +1,9 @@
+opcuaSession OPC1 opc.tcp://130.246.88.203:4840
+opcuaSubscription SUB1 OPC1 200
+
+opcuaOptions OPC1 sec-mode=None, sec-id="\Instrument\Settings\config\NDW2548\configurations\opcua\cert.txt", debug=10
+opcuaClientCertificate "C:\Instrument\Settings\config\NDW2548\configurations\opcua\client_cert.der", "C:\Instrument\Settings\config\NDW2548\configurations\opcua\client_private_key.pem"
```

Comment:
Can you use the `ICPCONFIGROOT` macro or similar here, so that we're not hard-coding machine names?